src/misc/LoadingLine/LoadingLine.mdx

```diff
@@ -0,0 +1,51 @@
+---
+name: LoadingLine
+route: /LoadingLine
+menu: Misc
+---
+
+import { Playground, Props } from 'docz';
+import { LoadingLine } from './LoadingLine';
+import * as dataLoading from '../../../assets/loadingView/dataLoading.json';
```

Comment:
Could you please change the folder's name `loadingView` by `loadingline` to match with the component and not the previous view anymore?